prow/pjutil/pjutil_test.go

```diff
@@ -750,10 +750,10 @@ func TestJobURL(t *testing.T) {
 					PathStrategy: prowapi.PathStrategyExplicit,
 				}},
 			}},
-			expected: "https://gubernator.com/build/bucket/pr-logs/pull/org_repo/1",
+			expected: "https://gubernator.com/build/gs/bucket/pr-logs/pull/org_repo/1",
```

Comment:
Should we provide a way to turn this off? When linking to spyglass we should prepend the storage provider, but other systems like gubernator may not support it.

AKA `https://gubernator.com/build/bucket/pr-logs/pull/org_repo/1` is still the correct URL here

Reply:
@fejta *push* :)

Reply:
Implemented